pyproject.toml

```diff
@@ -19,9 +19,9 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "3.10.*"
-Django = "^4.0.6"
-psycopg2 = "^2.9.3"
+python = "^3.6"
+Django = "^3.2"
```

Comment:
maybe in future also add django testing matrix as we have with CQRS?

Reply:
will keep it for the next version if you don't mind